src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {ScrollView, View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import ONYXKEYS from '../../../ONYXKEYS';
+import ROUTES from '../../../ROUTES';
+import NotFoundPage from '../../ErrorPage/NotFoundPage';
+import CardPreview from '../../../components/CardPreview';
+import HeaderWithBackButton from '../../../components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '../../../components/MenuItemWithTopDescription';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import assignedCardPropTypes from './assignedCardPropTypes';
+import useLocalize from '../../../hooks/useLocalize';
+import * as CurrencyUtils from '../../../libs/CurrencyUtils';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import * as CardUtils from '../../../libs/CardUtils';
+
+const propTypes = {
+    /* Onyx Props */
+    cardList: PropTypes.objectOf(assignedCardPropTypes),
+
+    /** Navigation route context info provided by react navigation */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** domain passed via route /settings/wallet/card/:domain */
+            domain: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    cardList: {},
+};
+
+function ExpensifyCardPage({
+    cardList,
+    route: {
+        params: {domain},
+    },
+}) {
+    const {translate} = useLocalize();
+    const domainCards = CardUtils.getDomainCards(cardList)[domain];
+    const virtualCard = _.find(domainCards, (card) => card.isVirtual) || {};
+    const physicalCard = _.find(domainCards, (card) => !card.isVirtual) || {};
+
+    if (_.isEmpty(virtualCard) || _.isEmpty(physicalCard)) {
+        return <NotFoundPage />;
+    }
```

Comment:
This should be an &&, since a user can have a physical card and not a virtual card (and vice-versa)

Reply:
changed to &&